server/db/orderItems.js

```diff
@@ -16,17 +16,31 @@ function addToOrder (userId, orderId, conn = connection) {
       if (!orderExists) {
         throw new Error('Order does not exist.')
       }
+    })
+    .then(() => {
       return users.getUser(userId, conn)
         .then(user => {
           if (!user) {
             throw new Error('User does not exist.')
```

Comment:
I missed this one in an earlier PR. See the next comment about throwing errors on the server side.